library/sql.inc

```diff
@@ -123,18 +123,121 @@ class ADODB_mysqli_log extends ADODB_mysqli
 
         return $this->genID;
     }
+
```

Comment:
Will migrate this custom ADODB_mysqli_log to a separate script called ADODB_mysqli_log.php